src/main/java/core/basesyntax/processdata/convertdata/ConvertDataImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.processdata.convertdata;
+
+import java.util.List;
+
+public class ConvertDataImpl implements ConvertData {
+    @Override
+    public List<List<String>> convertData(List<String> data) {
+        return data.stream()
+                .filter(s -> s.charAt(0) == ' ')
+                .map(String::trim)
+                .map(s -> List.of(s.split(",")))
+                .toList();

```

Comment:
Sorry, but I can't understand this code. Why do we need this filter here, why do we filter only those lines that begin with an empty character. This code is not very intuitive. And you return a list of lists of strings. Why not create a separate entity for the transaction and parse it here and return a list of transactions, as shown in the example for the task.

Reply:
Cause i receive data from file like 

```
columnName, columnName
      data,value
      data,value
```

I need to delete only first line so i found that it hasn`t space at the beginning, but i agree that it is not readable.